doc/man/taskrc.5.in

```diff
@@ -508,15 +508,6 @@ weekly recurring task is added with a due date of tomorrow, and recurrence.limit
 is set to 2, then a report will list 2 pending recurring tasks, one for tomorrow,
 and one for a week from tomorrow.
 
-.TP
-.B undo.style=side
-When the 'undo' command is run, Taskwarrior presents a before and after
-comparison of the data. This can be in either the 'side' style, which compares
-values side-by-side in a table, or 'diff' style, which uses a format similar to
-the 'diff' command.
```

Comment:
I take it that you think implementing the side style is impractical? Is there anyplace appropriate to print a warning if `undo.style` is configured?

Reply:
Style support hsa been gone for quite a while now, so my thinking was to just remove it completely.

We have three formats now (diff, side, and the current operation-based). I hope folks aren't parsing any of those directly. So I think just having one, somewhat-human-readable format is reasonable.

Reply:
Yeah, I agree that the current format is reasonable and an improvement, I just wondered if there were any explicit steps we should take here since the feature is implicitly transitioning from "to do" to "unplanned".

Reply:
I'll put it in the release notes. Also, `task show` will show the config as unknown, which will be a nice hint without being too loud about it.